src/components/Book.js

```diff
@@ -1,30 +1,91 @@
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { removeBook, deleteBook } from '../redux/books/booksSlice';
+import 'react-circular-progressbar/dist/styles.css';
+import './styles/Book.css';
 
-const Book = ({ title, author, id }) => {
+const Book = ({
+  title, author, category, id,
+}) => {
   const dispatch = useDispatch();
+  const progress = Math.floor(Math.random() * 100);
+  const chapter = Math.floor(Math.random() * 15);
   return (
     <div className="book">
-      <h2>{title}</h2>
-      <p>{author}</p>
-      <button
-        type="button"
-        className="remove-button"
-        onClick={() => {
-          dispatch(deleteBook(id));
-          dispatch(removeBook({ id }));
-        }}
-      >
-        Remove
-      </button>
+      <div className="info">
+        <p className="category">{category}</p>
+        <h2 className="title">{title}</h2>
+        <p className="author">{author}</p>
+        <div className="all-btns">
+          <button
+            className="btn-action"
+            type="button"
+            onClick={
+              () => {
+                dispatch(removeBook({ id }));
+                dispatch(deleteBook(id));
+              }
+            }
+          >
+            Remove
+          </button>
+          <span className="separator">|</span>
+          <button
+            className="btn-action"
+            type="button"
+            title="Comment"
+          >
+            Comment
+          </button>
+          <span className="separator">|</span>
+          <button
+            className="btn-action"
+            type="button"
+            title="Edit"
+          >
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="progress">
+        <div className="progressCircle">
+          <CircularProgressbar value={progress} />
+        </div>
+        <div className="progress-percentage">
+          <p className="percentage">
+            {progress}
+            %
+          </p>
+          <p className="progress-text">Completed</p>
+        </div>
+        <span className="separator2" />
+        <div className="progressInfo">
+          <p className="currentChapter">CURRENT CHAPTER</p>
+          <p className="chapter">
+
+            CHAPTER
+            {' '}
+            {chapter}
+
+          </p>
+          <button
+            className="update-progress"
+            type="button"
+            title="UPDATE PROGRESS"
+          >
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly improve the styling of the book card to match the design. As noted in the requirement we are expected to make an EXACT copy of the Zeplin design.

  - Kindly use the correct fonts. Some of your fonts do not match the ones in the design
  - Kindly note the spacing between the elements as well, for example, the right section of the card containing the "Update Progress" button should be moved more toward the left to give some space from the right border.
  - Kindly also add some side padding the button.

  Zeplin design:
  ![iosap](https://user-images.githubusercontent.com/81468675/230596999-fe14b39f-fe25-4033-93bd-51e96cb54fe3.jpg)

  <hr/>

  Your design:
  ![sjaia](https://user-images.githubusercontent.com/81468675/230597006-95bc019c-7f62-4d29-a8e6-f39bf7a3628e.jpg)